components/button/config.less

```diff
@@ -2,6 +2,7 @@
 @import "../input/config";
 @import "../border/config";
 
+@button-font-family: "effra", serif;
```

Comment:
Also, it should be set to the `@font-family-x` variable (`@font-family-sans-serif`, etc)